src/Administration/Resources/app/administration/src/module/sw-customer/page/sw-customer-create/index.js

```diff
@@ -160,6 +162,17 @@ Component.register('sw-customer-create', {
                     this.customerRepository.save(this.customer).then(() => {
                         this.isLoading = false;
                         this.isSaveSuccessful = true;
+
+                        const newCustomerNumber = this.customer.customerNumber;
+
+                        if (oldCustomerNumber !== newCustomerNumber) {
+                            this.createNotificationInfo({
```

Comment:
can you add a jest test that verifies that the notification is shown if the customer number changes?

Reply:
One test this by mocking the mixin, right?

Reply:
exactly, just making sure createNotififcation is only called when the customer number changed